### PR TITLE
Price API has been implemented for get the coin price dynamically

### PR DESCRIPTION
**Pull request by Shaharyar-saleem**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -106,7 +106,8 @@
                             <div class="col-6 col-lg-2 col-md-3 text-center color-white">
                                 <div class="coin-value-card" data-Speed="3000" data-aos="flip-right"
                                     data-aos-once="ture">
-                                    <h5 class="">0.00000000753379</h5>
+                                    <h5 class="" id="coin-price"></h5>
+                                    <!-- 0.00000000753379 -->
```
Comment (bobachef) [inline index.html:110]: remove this comment

File: index.html [hunk 2]
```diff
@@ -326,6 +327,9 @@ <h2>Road map</h2>
     <!-- animated number counter script -->
     <!-- <script src="./js/multi-animated-counter.js" defer></script> -->
```
Comment (bobachef) [inline index.html:328]: lets get rid of these comments too

File: index.html [hunk 3]
```diff
@@ -341,6 +345,17 @@ <h2>Road map</h2>
     <script>
         AOS.init();
     </script>
+
+    <!-- API fetch data -->
+    <script>
+        fetch('https://api.dex.guru/v1/tokens/0x3504de9e61fdff2fc70f5cc8a6d1ee493434c1aa-bsc')
```
Comment (bobachef) [inline index.html:351]: can we move this into it's own js file? we can have all of the stats stuff in that file.

File: index.html [hunk 4]
```diff
@@ -341,6 +345,17 @@ <h2>Road map</h2>
     <script>
         AOS.init();
     </script>
+
+    <!-- API fetch data -->
+    <script>
+        fetch('https://api.dex.guru/v1/tokens/0x3504de9e61fdff2fc70f5cc8a6d1ee493434c1aa-bsc')
+        .then(res =>  res.json())
+        .then(data =>{
+            const price = parseFloat(data.priceUSD, 15).toFixed(14);
```
Comment (bobachef) [inline index.html:354]: parseFloat only takes 1 parameter... please remove 15

File: index.html [hunk 5]
```diff
@@ -341,6 +345,17 @@ <h2>Road map</h2>
...
         AOS.init();
     </script>
+
+    <!-- API fetch data -->
+    <script>
+        fetch('https://api.dex.guru/v1/tokens/0x3504de9e61fdff2fc70f5cc8a6d1ee493434c1aa-bsc')
+        .then(res =>  res.json())
+        .then(data =>{
+            const price = parseFloat(data.priceUSD, 15).toFixed(14);
+            console.log(price);
```
Comment (bobachef) [inline index.html:355]: always remove logs before submitting pull requests

**Review by bobachef — CHANGES_REQUESTED** [review]
some fixes needed

File: index.html (hunk 3 above)
Comment (Shaharyar-saleem) [reply index.html:351]: sure let me implement all these changes

File: index.html (hunk 5 above)
Comment (Shaharyar-saleem) [reply index.html:355]: done

File: index.html (hunk 2 above)
Comment (Shaharyar-saleem) [reply index.html:328]: we will have to wait until we get the real data

**Comment by bobachef** [comment]
Superceded by https://github.com/tesla-safe/tesla-safe-web/pull/11/

File: index.html (hunk 1 above)
Comment (Shaharyar-saleem) [reply index.html:110]: removed

**Pull request closed without merging**